api/services/AuthorizationPolicy/index.js

```diff
@@ -117,7 +118,7 @@ policy.include('calendarEventInvite', (p) => {
     ({ viewer, entity: { event, status }, action }) =>
       isOrganiser(event, viewer) ||
       isAdmin(viewer) ||
-      (isUser(viewer) && status === 'Requested' && action === ':create'),
+      (isMember(viewer) && action === ':create'),
```

Comment:
Since user cannot create  requests i changed into the `isMember()` function.
That `status === 'Requested'` is not useful because if it is admin or orgainizer or auto_accepting_requests we do not evene assign Requested in any point .so it will always return false.

Reply:
Check the comment above.